classes/class-briqpay-ajax.php

```diff
@@ -29,6 +29,7 @@ public static function add_ajax_events() {
 			'briqpay_wc_log_js'                => true,
 			'briqpay_wc_update_checkout'       => true,
 			'briqpay_wc_change_payment_method' => true,
+			'update_order_orm'                 => true,
```

Comment:
This should be false. The true here is for the WordPress flag "nopriv" which means that if true, the AJAX can be triggered from a non-logged in account. Since this is 100% a function for an admin, they will always be logged in, and should due to that never be available for a non logged in user.